lib/std/ascii.zig

```diff
@@ -335,6 +335,24 @@ test "eqlIgnoreCase" {
     std.testing.expect(!eqlIgnoreCase("hElLo!", "helro!"));
 }
 
+pub fn startsWithIgnoreCase(haystack: []const u8, needle: []const u8) bool {
+    return if (needle.len > haystack.len) false else eqlIgnoreCase(haystack[0..needle.len], needle);
+}
+
+test "mem.startsWith" {
```

Comment:
I think this would be a bit more meaningful for a test case name, wouldn't you agree?

```suggestion
test "ascii.startsWithIgnoreCase" {
```

Reply:
Indeed, I did copy&paste the test cases from `mem.zig` and forgot to change the names.

Reply:
the prefix is actually redundant, the full test name ends up getting prefixed with ascii anyway since it's in ascii.zig. But it's harmless, no need to go through another round of changes IMO